recipe/meta.yaml

```diff
@@ -44,6 +44,8 @@ test:
   # https://github.com/conda-forge/staged-recipes/pull/292#issuecomment-208080893.
   commands:                # [not win]
     - nosetests zmq.tests  # [not win]
+  imports:                 # [win]
+    - zmq                  # [win]
```

Comment:
It looks like this is getting imported on all platforms above. Or am I missing something?